middle_end/clambda_primitives.ml

```diff
@@ -210,4 +213,5 @@ let result_layout (p : primitive) =
   match p with
   | Punbox_float -> Lambda.Punboxed_float
   | Punbox_int bi -> Lambda.Punboxed_int bi
+  | Pufloatfield _ -> Lambda.Punboxed_float
   | _ -> Lambda.layout_any_value
```

Comment:
Yes (indeed, my foot was wounded here in the preparation of this PR).  I didn't fix it because I saw the file also exists upstream, but that is dumb - this function surely does not, and anyway the diffs would be worth it here.  Fixed.